qmlbind/src/metaobject.cpp

```diff
@@ -6,6 +6,25 @@
 
 namespace QmlBind {
 
+// See qmetaobject_p.h for flag details
+
+enum PropertyFlags {
```

Comment:
This was more than I excepcted that could be done here, this makes me able to reason nicely about the code even without learning the details of `qobject_p.h`. :+1: